src/WPML_OptionsManager.php

```diff
@@ -537,7 +537,23 @@ public function _LogMenu() {
                 $emailLogList->display();
                 ?>
             </form>
-            <?php $this->displayMP3Banner() ?>
+			<?php
+			$banner = $this->displayMP3Banner();
```

Comment:
The `$this->displayMP3Banner()` call could be moved within the `if` section, roughly as follows:
```
$display = apply_filters('wpml_banner_display', true);
if ($display) {
    $this->displayMP3Banner();
}
```